### PR TITLE
[t130087] MIDE DEV V15 Migration - Operating Unit (OCA & BT) - Part 1

### PR DESCRIPTION
**Pull request by BT-apadron**
<div> Links to Odoo: </div> <ul>
<li><a target="_blank" href="https://odoo.bt-group.com/web#view_type=form&model=project.task&id=130087">[t130087] MIDE DEV V15 Migration - Operating Unit (OCA & BT) - Part 1</a></li>
</ul>
<div>Affected Modules:</div>
<table><thead><tr><th>Module</th><th>Ext</th></tr></thead>
<tr><td>sale_operating_unit</td><td>.pot, .py, .html, .rst, .png, .xml, .po, .txt</td></tr>
<tr><td>account_operating_unit</td><td>.pot, .py, .html, .rst, .png, .xml</td></tr>
<tr><td>sale_stock_operating_unit</td><td>.pot, .py, .html, .rst, .png, .po, .txt</td></tr>
<tr><td>stock_landed_cost_operating_unit</td><td>.rst, .xml, .py, .png</td></tr>
<tr><td>stock_account_operating_unit</td><td>.pot, .py, .html, .rst, .png, .po, .txt</td></tr>
</tbody></table>

### CODE REVIEW

**Comment by BT-apadron** [comment]
@BT-jmontero There is a dependency with module **stock_operanting_unit**. This one will be included in the task [t130202]
https://github.com/brain-tec/operating-unit/pull/343

File: sale_stock_operating_unit/README.rst [hunk 1]
```diff
@@ -7,23 +7,14 @@ Operating Unit in Sales Stock
...
-.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
-    :target: https://translation.odoo-community.org/projects/operating-unit-12-0/operating-unit-12-0-sale_stock_operating_unit
-    :alt: Translate me on Weblate
-.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
-    :target: https://runbot.odoo-community.org/runbot/213/12.0
-    :alt: Try me on Runbot
-
-|badge1| |badge2| |badge3| |badge4| |badge5| 
+.. |badge2| image:: https://img.shields.io/badge/github-minditp-lightgray.png?logo=github
+    :target: https://github.com/brain-tec/minditp/tree/15.0-mig-sale_stock_operating_unit/sale_stock_operating_unit
```
Comment (BT-tmeichtry) [inline sale_stock_operating_unit/README.rst]: I think you should use as branch for the OCA readme generator the branch "15.0.project_MI_465".
At the moment you are using the branch 15.0-mig-sale_stock_operating_unit but this branch will be deleted after the migration. Therefore you should always use the prod branch.

File: sale_stock_operating_unit/__manifest__.py [hunk 2]
```diff
@@ -6,10 +6,10 @@
 {
     "name": "Operating Unit in Sales Stock",
     "summary": "An operating unit (OU) is an organizational entity part of a "
-               "company",
```
Comment (BT-tmeichtry) [inline sale_stock_operating_unit/__manifest__.py]: Do you have a reason why you remove this empty spaces?
I think in my opinion the readability is better with the spaces.

Comment (BT-apadron) [reply sale_stock_operating_unit/__manifest__.py]: The pre-commit failed if I put this empty spaces.
image

Comment (BT-tmeichtry) [reply sale_stock_operating_unit/__manifest__.py]: Thanks for the info. I don't love it without spaces. But of course you don't need to change it.

File: sale_stock_operating_unit/README.rst (hunk 1 above)
Comment (BT-apadron) [reply sale_stock_operating_unit/README.rst]: Done correction in the commit [639a0b4](https://github.com/brain-tec/operating-unit/pull/341/commits/639a0b43d35cb3afa7c0ebc0fd9660db378ef3ee)

Comment (BT-tmeichtry) [reply sale_stock_operating_unit/README.rst]: Now it looks good to me!

**Pull request merged**